src/plugin.ts

```diff
@@ -30,3 +36,48 @@ export const XkcdComicCard =
             },
         }),
     );
+
+export const XkcdComicHomePageComponent = homePlugin.provide(
+    createCardExtension<XkcdComicProps>({
+        name: 'XkcdComicCard',
+        title: 'xkcd',
+        components: () => import('./components/XkcdComicCardHomePage'),
+        layout: {
+            height: {minRows: 1},
+            width: {minColumns: 3},
+        },
+        description: 'xkcd comic',
+        settings: {
+            schema: {
+                title: 'xkcd',
+                type: 'object',
+                properties: {
+                    showNav: {
+                        title: 'Show Navigation',
+                        type: 'boolean',
+                        default: true,
+                        description: 'Show navigation buttons'
+                    },
+                    showExplain: {
+                        title: 'Show Explain',
+                        type: 'boolean',
+                        default: true,
+                        description: 'Show an external link to xkcd wiki'
+                    },
+                    comicNumber: {
+                        title: 'Show specific comic number',
+                        type: 'number',
+                        default: LAST_INDEX,
+                        description: 'Show specific comic number. Default - show the last released comic'
+                    },
+                    proxyUrl: {
```

Comment:
This probably should not be in the settings but instead you should add this to README how to set up the proxy properly. This is because the users of the Backstage application might no have a clue about the configuration.

Reply:
You are right - this should not be available for regular user. I decided to remove support also for `useProxy` because of this https://dev.to/will_yama/how-to-overcome-cors-errors-2nh9 (long story short, it cannot work without some proxy because of CORS).